ci.properties

```diff
@@ -5,8 +5,8 @@ FAB_BASEIMAGE_VERSION=0.4.15
 # Pull below list of images from Hyperledger Docker Hub
 FAB_THIRDPARTY_IMAGES_LIST=kafka zookeeper couchdb
 # Set compatible go version
-GO_VER=1.12.9
+GO_VER=1.12.5
```

Comment:
@btl5037   Should this be 1.12.9 as well?

Reply:
No, our nightly release jobs still depend on this. This is actually what this PR is fixing